pulseapi/entries/test_views.py

```diff
@@ -380,6 +380,115 @@ def test_bookmarked_entries_view(self):
         self.assertEqual(len(bookmarkJson['results']), 1)
         self.assertEqual(id, bookmarkJson['results'][0]['id'])
 
+    def test_post_bookmark_entries_without_login(self):
+        """
+        Verify that anonymous users cannot bookmark a list of entries.
+        """
+
+        # get a legal entry and its associated id
+        entries = Entry.objects.all()
+        entry = entries[0]
+        id = entry.id
+
+        # ensure the user is logged out, then try to bookmark
+        self.client.logout()
+        url = '/api/pulse/entries/bookmarks/?ids=' + str(id)
+        postresponse = self.client.post(url)
+        self.assertEqual(postresponse.status_code, 403)
+
+        # verify bookmark count is zero
+        bookmarks = entry.bookmarked_by.count()
+        self.assertEqual(bookmarks, 0)
+
+    def test_post_bookmark_entries_with_login(self):
```

Comment:
What is being tested in this test? For toggling bookmarks we already have `/entries/:id/bookmark` so if this simply tests that bulk bookmarking but with only a single entry works, then we should combine that with posting a comma delimited list.